AdobeStockImageAdminUi/Test/Unit/Ui/Component/Listing/Filter/ColorTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImageAdminUi\Test\Unit\Model;
+
+use Magento\AdobeStockImageAdminUi\Ui\Component\Listing\Filter\Color;
+use Magento\Framework\Api\FilterBuilder;
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+use Magento\Framework\View\Element\UiComponentFactory;
+use Magento\Ui\Component\Filters\FilterModifier;
+use Magento\Ui\Model\ColorPicker\ColorModesProvider;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * SignInConfigProviderTest test.
+ */
+class ColorTest extends TestCase
+{
+    /**
+     * @var MockObject|ContextInterface $contextInterface
+     */
+    private $contextInterface;
+
+    /**
+     * @var MockObject|UiComponentFactory $uiComponentFactory
+     */
+    private $uiComponentFactory;
+
+    /**
+     * @var MockObject|FilterBuilder $filterBuilder
+     */
+    private $filterBuilder;
+
+    /**
+     * @var MockObject|FilterModifier $filterModifier
+     */
+    private $filterModifier;
+
+    /**
+     * @var MockObject|ColorModesProvider $colorModesProvider
+     */
+    private $colorModesProvider;
+
+    /**
+     * @var Color
+     */
+    private $color;
+
+    /**
+     * Set Up
+     */
+    protected function setUp()
+    {
+        $this->contextInterface = $this->createMock(ContextInterface::class);
+        $this->uiComponentFactory = $this->createMock(UiComponentFactory::class);
+        $this->filterBuilder = $this->createMock(FilterBuilder::class);
+        $this->filterModifier = $this->createMock(FilterModifier::class);
+        $this->colorModesProvider = $this->createMock(ColorModesProvider::class);
+
+        $this->color = new Color(
+            $this->contextInterface,
+            $this->uiComponentFactory,
+            $this->filterBuilder,
+            $this->filterModifier,
+            $this->colorModesProvider,
+            [],
+            ['name' => 'name_one', 'config/colorFormat' => 'format_one']
+        );
+
+        $reflection = new \ReflectionClass($this->color);
```

Comment:
Please pass filterData by mocking `Context->getFiltersParams` method instead of modifying the test class with reflection.

Reply:
Ok